Contributing.md

```diff
@@ -0,0 +1,95 @@
+
+## How to Start Contributing
+The KERI community welcomes [all kinds of contributions](https://opensource.guide/how-to-contribute/).
+
+For simple contributions like fixing typos sometimes you can just submit a naked PR with a title like "I'm fixing a spelling issue in x,y,z".  If these PRs are small often times they'll just get merged without further review.  Typos or fixing names that were mistakenly applied are human nature.
+
+A really simple way to contribute is just to spread the word.  Let people know about KERI, create a welcoming community to newcomers by actively participating and helping with what you know, and making cools things and talking about them is helpful to grow.
+
+Issues and discussions are meant to be discussed, commented upon.  Even a +1 to an issue that you're also experiencing can help the core maintainers decide where to focus their energies.  Discussions can help provide clarity if things aren't exactly clear.  Feel free to contribute (but also note the homework in [PREREQUISITES](#prerequisites), its frustrating for people to explain things that are already explained elsewhere).
+
+However, this particular file is more about contributing code.  The general principal in this (and all open source really) is if you can think of a way that things might be better, instead of just suggesting it or discussing it, often times its a good idea to do some work to show what/how/when an idea might look like or contributing code/scripts/tests/issues.  Code >> discussion 9 times out of ten.  Balanced of course with the caveat not to go off into the ivory tower for months and do lots of work before presenting your work and finding out that the community doesn't like it.  Small prototypes/examples are often best.  The rest of the details of contributing code are below.
+
+## Contribution Guidelines
+
+### Code Style/Conventions
+[PythonStyleGuide](./ref/PythonStyleGuide.md)  
+
+### Commit Message Guidelines
+There are no hard and fast guidelines but it is helpful to:
+- Descriptive, to the point messages are ideal
+- Link to the issue you're trying to solve with a message of how the commit does or doesn't solve that issue.
+- Be sure to note important issues for checkpoint commits like "does not build"
+- Most importantly, the commit message should explain the WHY of all the code changes.  When reviewing, reviewers will be confused if code changes for things you didn't list in the commit aren't there.
+
+### Branching Strategy
+Git flow.  Branch from development, name your feature branches something like `feature-name-of-my-feature` and bugs something like `bug-bug-name` and link the github issue that you (or someone else) def should have created for most non-trivial bugs.  When you have fully implemented or fixed, submit a PR to WebOfTrust/keripy.  If you need feedback it can also be appropriate to submit a **draft** PR to this repository.
```

Comment:
I don't think we need to bring GitFlow into it, I think your description is enough.

Reply:
Good point.  Will take it out.